deploy/helm/csi-wekafsplugin/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Default values for csi-wekafs-driver.
+
+replicaCount: 1
+instanceCount: 1
```

Comment:
why in values if it's not something that supposed to change?

Reply:
Maybe they want to change it for redundancy?